FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/organisationid/update/UpdateOrganisationIdRequest.java

```diff
@@ -0,0 +1,96 @@
+package com.verisec.frejaeid.client.beans.organisationid.update;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.verisec.frejaeid.client.beans.common.RelyingPartyRequest;
+import com.verisec.frejaeid.client.beans.general.OrganisationIdAttribute;
+
+import java.util.List;
+import java.util.Objects;
+
+public class UpdateOrganisationIdRequest implements RelyingPartyRequest {
+
+    private final String identifier;
+    private final List<OrganisationIdAttribute> additionalAttributes;
+    private final String relyingPartyId;
+
+    public static UpdateOrganisationIdRequest create(String identifier, List<OrganisationIdAttribute> additionalAttributes){
+        return new UpdateOrganisationIdRequest(identifier, additionalAttributes);
+    }
+
+    public static UpdateOrganisationIdRequest create(String identifier, List<OrganisationIdAttribute> additionalAttributes, String relyingPartyId){
+        return new UpdateOrganisationIdRequest(identifier, additionalAttributes, relyingPartyId);
+    }
+
+    /**
+     * Returns instance of {@linkplain UpdateOrganisationIdRequest}
+     *
+     * @param identifier           identifier to be updated for the end user. It cannot be
+     *                             {@code null} or empty.
+     * @param additionalAttributes additional attributes related to the identifier.
+     *                             It can be {@code null} or empty.
+     * @return request
+     */
+    @JsonCreator
+    private UpdateOrganisationIdRequest(@JsonProperty("identifier") String identifier,
```

Comment:
You don't need 2 private constructors, just use one bellow. Check whether it needs to have jackson annotations.